tests/Microsoft.Identity.Test.Integration/SeleniumTests/DeviceCodeFlowIntegrationTest.cs

```diff
@@ -95,21 +97,57 @@ public async Task DeviceCodeFlowTestAsync()
 
             Assert.IsNotNull(result);
             Assert.IsTrue(!string.IsNullOrEmpty(result.AccessToken));
-        }
+        }
+
+
+        /*[TestMethod]
```

Comment:
>/* [](start = 8, length = 2)

nit: if you remove the TestMethod the test will not run. Perhaps you should add the Ignore attribute instead?